### ISSUE TITLE
Security vulnerabilities in debian image

### ISSUE THREAD

**Issue by dohnto** [nicolargo/glances#1880]
First of all, thanks for this beautiful software!

**Is your feature request related to a problem? Please describe.**

When using [trivy](https://aquasecurity.github.io/trivy/v0.18.3/) to scan the docker images for security vulnerabilities I can see following results
```
$ trivy image nicolargo/glances:3.1.7
...
nicolargo/glances:3.1.7 (debian 10.9)
=====================================
Total: 556 (UNKNOWN: 2, LOW: 412, MEDIUM: 55, HIGH: 80, CRITICAL: 7)
...
```

On the other hand
```
$ trivy image nicolargo/glances:latest-alpine
...
nicolargo/glances:latest-alpine (alpine 3.13.5)
===============================================
Total: 2 (UNKNOWN: 0, LOW: 1, MEDIUM: 0, HIGH: 1, CRITICAL: 0)
...
```

The number of CVEs included in the debian version is very high.

My motivation is to produce secure software and to comply with security processes. It seems, that alpine one is doing better job, but unfortunately, since it is not versioned, I cannot rely on using latest (I might consider using docker @digest ).

**Describe the solution you'd like**
My question is how much is the alpine version maintainted and if it is, can we start to version it in similar fashion we do the debian one?

Is there any alpine image that corresponds to current latest release version (3.1.7)?

**Describe alternatives you've considered**

Other solution would be to address the CVEs in the debian image, ...

Thanks for any input on this problem.

**Comment by nicolargo** [comment]
Should be corrected when next release (Glances 3.2) will be release.

**Comment by RazCrimson** [comment]
Debian images are dropped now.
